Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -25,16 +25,17 @@ namespace winrt::BabylonReactNative::implementation {
                     static_cast<uint32_t>(Windows::UI::Core::CoreInputDeviceTypes::Mouse) |
                     static_cast<uint32_t>(Windows::UI::Core::CoreInputDeviceTypes::Touch) |
                     static_cast<uint32_t>(Windows::UI::Core::CoreInputDeviceTypes::Pen));
-                auto coreInput = trueThis->CreateCoreIndependentInputSource(deviceTypes);
+                trueThis->_inputSource = trueThis->CreateCoreIndependentInputSource(deviceTypes);

```

Comment:
I guess it is not possible to get the `CreateCoreIndependentInputSource` from the event handlers?

Reply:
we initially prototyped this to have _inputSource updates run on their own thread outside the threadpool which required access to the _inputSource outside the scope of callbacks. It may be overkill to hold this reference for now, but we probably will move this logic outside the thread pool long term when we can successfully tear down the EngineView/when other graphics life cycle bugs are fixed.